<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogGlobalConfig</class>
 <widget class="QDialog" name="DialogGlobalConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>全局配置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>全局配置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0,0,0,1,0">
      <item row="0" column="0">
       <widget class="QLabel" name="wheelIdentifyLable_2">
        <property name="text">
         <string>功能开关</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="plcOpen">
        <property name="text">
         <string>PLC行车状态响应</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="judge">
        <property name="text">
         <string>判伤开关</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="autoReport">
        <property name="text">
         <string>自动报表</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="wheelIdentifyLabel">
        <property name="text">
         <string>车轮识别参数</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="rate">
        <property name="prefix">
         <string>倍率 </string>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="radius">
        <property name="prefix">
         <string>半径 </string>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="average">
        <property name="prefix">
         <string>平均 </string>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QSpinBox" name="standard">
        <property name="prefix">
         <string>评分 </string>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="templatePathLabel">
        <property name="text">
         <string>报表模板路径</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="5">
       <widget class="QLineEdit" name="templatePath">
        <property name="toolTip">
         <string>车辆扫查数据的保存路径, 支持自动格式化.
{time} 日期和时间
{train} 车号</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QPushButton" name="browseTemplate">
        <property name="text">
         <string>浏览...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="documentPathTitle">
        <property name="text">
         <string>数据保存路径</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="5">
       <widget class="QLineEdit" name="documentPath">
        <property name="toolTip">
         <string>车辆扫查数据的保存路径, 支持自动格式化.
{time} 日期和时间
{train} 车号</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QPushButton" name="browseDocument">
        <property name="text">
         <string>浏览...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxChannelGroups">
     <property name="title">
      <string>探头分组配置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="channelGroups"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="addChannelGroup">
          <property name="icon">
           <iconset resource="../../Source/plugins/Core/Icons.qrc">
            <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="removeChannelGroup">
          <property name="icon">
           <iconset resource="../../Source/plugins/Core/Icons.qrc">
            <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBoxWheelSetLayout">
     <property name="title">
      <string>轮图配置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="wheelSetLayouts"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QPushButton" name="clear">
          <property name="text">
           <string>清除选中</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="addLayout">
          <property name="icon">
           <iconset resource="../../Source/plugins/Core/Icons.qrc">
            <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="removeLayout">
          <property name="icon">
           <iconset resource="../../Source/plugins/Core/Icons.qrc">
            <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxChannelConfig">
     <property name="title">
      <string>通道配置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QTreeView" name="channels">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelRaw">
          <property name="text">
           <string>源</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="channelRaw">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>10000</number>
          </property>
          <property name="maximum">
           <number>30000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelStart">
          <property name="text">
           <string>起始</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="channelStart">
          <property name="keyboardTracking">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <number>10000</number>
          </property>
          <property name="maximum">
           <number>30000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelEnd">
          <property name="text">
           <string>末尾</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="channelEnd">
          <property name="keyboardTracking">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <number>10000</number>
          </property>
          <property name="maximum">
           <number>30000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSpace">
          <property name="text">
           <string>间隔</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="channelSpace">
          <property name="keyboardTracking">
           <bool>true</bool>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="copy">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>复制</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="load">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>加载</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="add">
          <property name="icon">
           <iconset resource="../../Source/plugins/Core/Icons.qrc">
            <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="remove">
          <property name="icon">
           <iconset resource="../../Source/plugins/Core/Icons.qrc">
            <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Source/plugins/Core/Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogGlobalConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogGlobalConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
